pkgs/misc/vscode-extensions/default.nix

```diff
@@ -653,6 +653,23 @@ let
         };
       };
 
+      Ionide.Ionide-fsharp = buildVscodeMarketplaceExtension {
```

Comment:
```suggestion
      ionide.ionide-fsharp = buildVscodeMarketplaceExtension {
```

Reply:
rather than having mixed-case, at the moment vscode extensions is trying to uniformize to smallcase.

Reply:
It should be fixed now. Thanks!